.travis.yml

```diff
@@ -8,13 +8,13 @@ cache:
 install:
   - nvm install 6
   - npm install
+  - npm run build
   - pip install -r requirements/dev.txt
   - pip install coveralls
 script:
   - py.test --cov
   - python manage.py makemigrations --dry-run --check --noinput
   - ./node_modules/.bin/polylint -F
-  - npm run build
```

Comment:
Leave it like this, but i am unsure:
the name of the command implies that he somehow builds something with the source code, then it should go in the script section. But since what he actually does is download a lot of JS stuff, leave it in install

Reply:
I don't care where it goes, just wanted to have it in the same section in all projects.

Reply:
ok great than it should be there.